README.md

```diff
@@ -1,17 +1,29 @@
 # Morpho Token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradability
```

Comment:
```suggestion
### Upgradeability
```

Reply:
I don't have access. Can you do a screenshot?

<img width="492" alt="image" src="https://github.com/user-attachments/assets/9a80db4d-ad2d-41ec-b121-f86072558710">

Reply:
Since both are ok, let's just stick to the naming of OZ, which is a lib that we are actually using in this contract

Reply:
This is really not important, but I actually don't think we should follow their choice here.